net/linuxptp/patches/002-honour-COPTS.patch

```diff
@@ -0,0 +1,11 @@
+--- a/makefile
++++ b/makefile
+@@ -20,7 +20,7 @@ KBUILD_OUTPUT =
+ DEBUG	=
+ CC	= $(CROSS_COMPILE)gcc
+ VER     = -DVER=$(version)
+-CFLAGS	= -Wall $(VER) $(incdefs) $(DEBUG) $(EXTRA_CFLAGS)
++CFLAGS	+= -Wall $(VER) $(incdefs) $(DEBUG) $(EXTRA_CFLAGS)
```

Comment:
We can avoid carrying an extra patch by `MAKE_VARS += EXTRA_CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS)"`